specs/xaml-backdrop-api.md

```diff
@@ -0,0 +1,316 @@
+
+        if(_micaController != null)
+        {
+            throw new Exception("This controller cannot be shared");
+        }
+
+        SetControllerConfig(connectedTarget, xamlRoot);
+
+        _micaController = new MicaController();
+        _micaController.AddSystemBackdropTarget(connectedTarget);
+    }
+
+    protected override void OnTargetDisconnected(ICompositionSupportsSystemBackdrop disconnectedTarget)
+    {
+        base.OnTargetDisconnected(disconnectedTarget);
+
+        _micaController.RemoveSystemBackdropTarget(disconnectedTarget);
+        _micaController = null;
+    }
+
+    void SetControllerConfig(ICompositionSupportsSystemBackdrop connectedTarget, XamlRoot xamlRoot)
+    {
+        var config = GetDefaultSystemBackdropConfiguration(connectedTarget, xamlRoot);
+        _micaController.SetSystemBackdropConfiguration(config);
+    }
+}
+```
+
+## SystemBackdrop.GetDefaultSystemBackdropConfiguration protected method
+
+Returns a
+[SystemBackdropConfiguration](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Composition.SystemBackdrops.SystemBackdropConfiguration)
+object that is set and maintained automatically, and can be passed to
+[ISystemBackdropControllerWithTargets.SetSystemBackdropConfiguration](https://docs.microsoft.com/windows/windows-app-sdk/api/winrt/Microsoft.UI.Composition.SystemBackdrops.ISystemBackdropControllerWithTargets.SetSystemBackdropConfiguration).
+
+The properties on `SystemBackdropConfiguration` you receive may change over time:
+
+* `Theme`: set based on the target element's
+[ElementTheme](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.ElementTheme)
+
+* `IsInputActive`: true if the target of the `SystemBackdrop` is in the active window.
+
+* `IsHighContrast`: true if the target of the `SystemBackdrop` is in high contrast mode.
+
+
+## SystemBackdrop.OnDefaultSystemBackdropConfigurationChanged protected method
+
+Override this method to be called when the object returned by `GetDefaultSystemBackdropConfiguration` has changed.
+
+This is useful if you're using a custom `SystemBackdropConfiguration`.
+The following example shows a `MicaBackdrop` similar to the previous example,
+but always forces the theme to be light.
+
+```cs
+protected override void OnSystemDefaultBackdropConfigurationChanged(SystemDefaultBackdropChangedEventArgs e)
+{
+    SetControllerConfig(e.ConnectedTarget, e.AssociatedRoot);
+}
+
+void SetControllerConfig(ICompositionSupportsSystemBackdrop connectedTarget, XamlRoot xamlRoot)
+{
+    var config = GetDefaultSystemBackdropConfiguration(connectedTarget, xamlRoot);
+    config.Theme = SystemBackdropTheme.Light;
+    _micaController.SetSystemBackdropConfiguration(config);
+}
+
+```
+
+
+
+## Other `SystemBackdrop` members
+
+| Member | Description |
+| - | - |
+| `OnTargetConnected` virtual method | Called when this object is attached to an object, for example when set on `Window.SystemBackdrop` |
```

Comment:
Is it possible to make these two method descriptions slightly more concrete? 'object' is used to name two different entities and in the first part of the sentence one is attached to the other, and in the second part of the sentence one is set on the other.

Reply:
Recommend: make this more clear

Reply:
I've been using the phrase  "appropriate container object"  (eg Window) for 2nd 'object' here. (If using this, also define this class above in Lines 81-85)